container/BUILD

```diff
@@ -71,6 +74,7 @@ py_binary(
     srcs_version = "PY2AND3",
     visibility = ["//visibility:public"],
     deps = [":build_tar_lib"],
+    legacy_create_init = True,
```

Comment:
You can add a TODO assigned to me (xingao) to try to flip this to False once we remove the dependency on @bazel_source (if that's the only reason that blocks flipping this bit.)

Reply:
Use Format

# TODO(xingao): .....

Reply:
Ok, added in latest commit. Yes, it appears that it's only the dependency on @bazel_source that requires that flag.